official/resnet/resnet.py

```diff
@@ -664,65 +665,20 @@ class ResnetArgParser(argparse.ArgumentParser):
   """
 
   def __init__(self, resnet_size_choices=None):
-    super(ResnetArgParser, self).__init__()
-    self.add_argument(
-        '--data_dir', type=str, default='/tmp/resnet_data',
-        help='The directory where the input data is stored.')
-
-    self.add_argument(
-        '--num_parallel_calls', type=int, default=5,
-        help='The number of records that are processed in parallel '
-        'during input processing. This can be optimized per data set but '
-        'for generally homogeneous data sets, should be approximately the '
-        'number of available CPU cores.')
+    super(ResnetArgParser, self).__init__(parents=[
+      official.utils.arg_parsers.LocationParser(data_dir=True, model_dir=True),
```

Comment:
The import logic seems off here-- shouldn't this be `from official.utils import arg_parsers` above, then just arg_parsers.... here?

Reply:
Yes, the import logic of official module is like @karmel mentioned.

Reply:
Hm, it looks like `from foo.bar import baz` is allowed by the style guide. I will certainly change this.